.github/workflows/ci.yaml

```
@@ -103,24 +95,24 @@ jobs:
         if: always()
         env:
           GH_TOKEN: ${{ github.token }}
-        run: echo "execution_time=$(gh run list --json createdAt,updatedAt -w ci.yaml -L 1 | jq -r '.[0] | (( (strtotime(.updatedAt) - strtotime(.createdAt)) ))')s" >> $GITHUB_ENV
+        run: echo "execution_time=$(gh run list --json createdAt,updatedAt | jq -r '.[0] | (.updatedAt | fromdateiso8601) - (.createdAt | fromdateiso8601)')" >> $GITHUB_ENV
 
       - name: Send email notification
         if: always()
         uses: dawidd6/action-send-mail@v3
         with:
-          server_address: ${{ inputs.mail_server_address }}
-          server_port: ${{ inputs.mail_server_port }}
+          server_address: ${{ secrets.mail_server_address }}
+          server_port: ${{ secrets.mail_server_port }}
           username: ${{ secrets.mail_username }}
           password: ${{ secrets.mail_password }}
           subject: ${{ inputs.mail_subject  }}
-          to: ${{ inputs.mail_receiver != '' && inputs.mail_receiver || env.DEFAULT_MAIL_RECEIVER }} 
+          to: ${{ secrets.mail_to != '' && secrets.mail_to || env.DEFAULT_MAIL_RECEIVER }}
```

Can you replace to this
```suggestion
          to: ${{ secrets.mail_to || env.DEFAULT_MAIL_RECEIVER }}
```

---

Done